Units/parser-javascript.r/js-crlf.d/expected.tags

```diff
@@ -0,0 +1,12 @@
+v1	input.js	/^var v1, v2$/;"	v
+v2	input.js	/^var v1, v2$/;"	v
+v3	input.js	/^var v3 = 1$/;"	v
+v4	input.js	/^,   v4 = 2$/;"	v
+v5	input.js	/^,   v5 = 3$/;"	v
+f1_c1_m1	input.js	/^    f1_c1_m1:$/;"	m	class:f1.f1_c1
+f1_c1_m2	input.js	/^    f1_c1_m2$/;"	m	class:f1.f1_c1
+f1_c1	input.js	/^  var f1_c1 = {$/;"	c	class:f1
```

Comment:
It seems that f1 is a function.

Reply:
Indeed. As mentioned somewhere it's a known limitation of the JS parser, with a `FIXME` in the code.